src/Database/DBA.php

```diff
@@ -420,7 +420,7 @@ public static function p($sql) {
 
 		if ((substr_count($sql, '?') != count($args)) && (count($args) > 0)) {
 			// Question: Should we continue or stop the query here?
-			Logger::log('Parameter mismatch. Query "'.$sql.'" - Parameters '.print_r($args, true), Logger::DEBUG);
+			Logger::error('Parameter mismatch. Query "'.$sql.'" - Parameters '.print_r($args, true));
```

Comment:
is this really an error? Debug is mapped to notice normally. Maybe you can fill the context array with the parameters and the query.

Reply:
Maybe more of a warning.